jaxrs/WFLY-12138-microprofile-rest-client-1.3.adoc

```diff
@@ -0,0 +1,53 @@
+= EAP7-1209 / WFLY-12138 Microprofile REST Client 1.3 support
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+This is about providing an implementation of the Microprofile REST Client 1.3.1 spec (https://github.com/eclipse/microprofile-rest-client/releases/tag/microprofile-rest-client-1.0).
```

Comment:
The link is an old tag, should be 1.3 instead of 1.0. However, there's already a new 1.3.2 release at https://github.com/eclipse/microprofile-rest-client/releases. Maybe we should wait for the final version available?

Reply:
Thanks for the feedback, I've updated the link to 1.3.1 tag. As far as I understand the spec is final, it's just the tck that has been fixed. I'd say let's go with what we have now, we can pull a newer version in future if relevant fixes are added.

Reply:
Right, just the TCK. However, we'll need those for testing so might need the fixes as well.

Reply:
OK, no problem, I've reworded the doc a bit, so that it's clear we'll try to use the latest 1.3.x version available (we're passing 1.3.2 currently, there's a pending PR against RESTEasy master for that).

Reply:
Thanks!